src/components/Text/index.module.css

```diff
@@ -34,12 +34,7 @@
   line-height: 28px;
 }
 
-@media (min-width: 950px) {
-  .root.small {
-    font-size: 16px;
-    line-height: 20px;
-  }
-
+@media (min-width: 768px) {
```

Comment:
Font-sizes have to change at the 950 pixels mark now, due to responsiveness issues.

Reply:
Nice catch, probably I did 💩 on the rebase, thanks